Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Local_User_Account_Creation.yml

```diff
@@ -0,0 +1,2395 @@
+          accessor: results.user_creation_count
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1591,
+          "y": 1449
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "66":
+    id: "66"
+    taskid: f3c6fe2d-cd09-4935-80bb-f0626194e6ef
+    type: title
+    task:
+      id: f3c6fe2d-cd09-4935-80bb-f0626194e6ef
+      version: -1
+      name: Creator User Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "71"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 786,
+          "y": 2526
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "67":
+    id: "67"
+    taskid: 4e06f579-a56a-447f-838c-7f5a98db85f3
+    type: regular
+    task:
+      id: 4e06f579-a56a-447f-838c-7f5a98db85f3
+      version: -1
+      name: Disable creator locally
+      description: Runs Powershell code on the affected host to disable the local user on the machine.
+      script: '|||core-run-script-execute-commands'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "43"
+    scriptarguments:
+      command_type:
+        simple: native
+      commands:
+        complex:
+          root: Core.OriginalAlert.event.action_evtlog_data_fields
+          accessor: SubjectUserName
+          transformers:
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: powershell -Command Disable-LocalUser -Name "
+              suffix:
+                value:
+                  simple: '"'
+      endpoint_ids:
+        complex:
+          root: alert
+          accessor: agentid
+          transformers:
+          - operator: uniq
+      is_raw_command:
+        simple: "true"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1186,
+          "y": 2804
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "70":
+    id: "70"
+    taskid: 18e9f289-19ea-4b1b-b1e3-034b89c56d0f
+    type: regular
+    task:
+      id: 18e9f289-19ea-4b1b-b1e3-034b89c56d0f
+      version: -1
+      name: Undetermined user type - manual
+      description: |-
+        Active Directory was unavailable, and user details for "${Core.OriginalAlert.event.action_evtlog_data_fields.SubjectUserName}" could not be retrieved.
```

Comment:
Based on the conditional task, the user could not be retrieved because the integration does not exist. We should mention that.

Reply:
This is displayed in the message of the manual task where it's explained